.gitremotes

```diff
@@ -0,0 +1,3 @@
+[remote "upstream"]
+	url = https://github.com/middlewares/utils.git
+	fetch = +refs/heads/*:refs/remotes/upstream/*
```

Comment:
this file is not needed, right? Can you remove it?

Reply:
Sorry, habit of mine to keep track of upstream repos in case they get renamed or disappear later - it's gone now